html/php-pre-challenge1/index.php

```diff
@@ -1,5 +1,17 @@
 <?php
 for ($i = 1; $i <= 100; $i++) {
     //ここからコードを書く
-    echo $i;
+    if($i % 3 === 0 && $i % 5 === 0){
```

Comment:
これでもよいですが、こう書くとCPUの計算を減らせます。
せっかくなので修正しましょう。
```suggestion
    if($i % 15 === 0){
```

Reply:
ここがまだ修正されてないですね。
This hasn't been fixed yet.

Reply:
ごめん。忘れた

> ここがまだ修正されてないですね。
> This hasn't been fixed yet.

ごめん。忘れた